independent-publisher-connectors/myStrom/apiDefinition.swagger.json

```diff
@@ -0,0 +1,640 @@
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "string",
+                        "description": "",
+                        "title": "Device ID"
+                      },
+                      "name": {
+                        "type": "string",
+                        "description": "",
+                        "title": "Name"
+                      },
+                      "power": {
+                        "type": "number",
+                        "format": "float",
+                        "description": "Current power consumption (W).",
+                        "title": "Power"
+                      },
+                      "type": {
+                        "type": "string",
+                        "description": "Device's type.",
+                        "title": "Type"
+                      },
+                      "state": {
+                        "type": "string",
+                        "description": "ON or OFF.",
+                        "title": "State"
+                      },
+                      "color": {
+                        "type": "string",
+                        "description": "In HSV format.",
+                        "title": "Color"
+                      },
+                      "temperature": {
+                        "type": "number",
+                        "format": "float",
+                        "description": "Current temperature (C).",
+                        "title": "Temperature"
+                      },
+                      "brightness": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "",
+                        "title": "Brightness"
+                      }
+                    }
+                  },
+                  "description": "devices"
+                }
+              }
+            }
+          },
+          "default": {
+            "description": "Default response",
+            "schema": {}
+          }
+        },
+        "summary": "Get devices",
+        "description": "Get list of all user's devices",
+        "operationId": "GetDevices",
+        "parameters": [
+          {
+            "name": "authToken",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "Authentication Token",
+            "x-ms-summary": "Authentication Token",
+            "format": "password",
+            "x-ms-visibility": "important"
+          }
+        ]
+      }
+    },
+    "/device/{Device ID}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "Response",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "id": {
+                  "type": "string",
+                  "description": "",
```

Comment:
@tposzytek Empty description values can be removed until unless you have specific reason

Reply:
Done